extensions/amp-iframe/amp-iframe.md

```diff
@@ -200,21 +200,21 @@ The iframe can listen to an `intersection` message from the parent window to rec
 
 ```javascript
 window.addEventListener('message', function(event) {
-  const listener = function(event) {
-    if (event.source != window.parent ||
-        event.origin != window.context.location.origin ||
-        !event.data ||
-        event.data.sentinel != 'amp' ||
-        event.data.type != 'intersection') {
-      return;
-    }
-    event.data.changes.forEach(function (change) {
-      console.log(change);
-    });
-  };
+  if (event.source != window.parent ||
+      event.origin == window.location.origin ||
+      !event.data ||
+      event.data.sentinel != 'amp' ||
+      event.data.type != 'intersection') {
+    return;
+  }
+  event.data.changes.forEach(function (change) {
+    console.log(change);
+  });
 });
 ```
 
+*Note: If script lives in nested iframe, change `window.parent` to the top AMP window instead*
```

Comment:
I moved this above the `postMessage` example, since there it should be top amp window as well. PTAL.